keps/sig-apps/3140-TimeZone-support-in-CronJob/README.md

```diff
@@ -203,38 +203,7 @@ None.
 
 #### GA
```

Comment:
are there any plans to add validation to apiserver that would forbid inserting `TZ` into the `Schedule` ?

https://github.com/kubernetes/kubernetes/blob/9787f46bbf05349ca53cd8b04cd2f7a76771646a/pkg/controller/cronjob/cronjob_controllerv2.go#L737